eslint.config.js

```
@@ -0,0 +1,101 @@
+import globals from "globals";
+import eslint from "@eslint/js";
+import tsdoc from "eslint-plugin-tsdoc";
+import jsdocPlugin from "eslint-plugin-jsdoc";
+import tseslint from "typescript-eslint";
+import prettierPlugin from "eslint-plugin-prettier";
+import eslintConfigPrettier from "eslint-config-prettier/flat";
+
+export default tseslint.config(
+  {
+    ignores: ["node_modules/", "dist/"],
```

ESLint

[error] Replace `"node_modules/",·"dist/"],` with `'node_modules/',·'dist/']`

(prettierPlugin/prettier)